core/station/api/spec.did

```diff
@@ -2362,6 +2394,14 @@ type ExternalCanisterChangeCallRequestPoliciesInput = variant {
   RemoveByPolicyIds : vec UUID;
   // Override the request policies for the specified execution methods.
   OverrideSpecifiedByExecutionMethods : vec ExternalCanisterCallRequestPolicyRuleInput;
+  // Override the request policies for the specified execution and validation method pairs.
+  OverrideSpecifiedByExecutionValidationMethodPairs : vec record {
+    // The method configuration that is used to represent the execution and validation method pair.
+    method_configuration : CanisterExecutionAndValidationMethodPair;
```

Comment:
Why do we need to perform the grouping explicitly in the args as opposed to deriving the grouping from the flattened `vec ExternalCanisterCallRequestPolicyRuleInput` as for `OverrideSpecifiedByExecutionMethods`?

Reply:
Mostly because this way we can also remove in the same request, with the above we need to make 2 separate requests, e.g. in the UI with this you can enable the user to rename a method & validation pair and you would simply make one request that would remove the renamed method and add the new one.

Reply:
> rename a method & validation pair

can't this be risky in case there already exists a permission/policy for the new pair?

Reply:
I would say it’s up for the approver to check and block, on the UI if there’s another one we don’t let the user submit that request, this provides a better UX and helps avoid submitting an override request when there’s already another entry

Reply:
> Mostly because this way we can also remove in the same request

Doesn't that mean that we should group by execution method in the request in the other case, too, instead of providing a flattened list?

Reply:
correct, that would make it consistent, i'll make the change.